src/utils/validateForm.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { fromJS } from 'immutable';
+import { FormattedMessage } from 'react-intl';
+
+import messages from '../components/WizardForm/messages';
+
+const validate = values => {
+  const errors = {};
+
+  if (!fromJS(values).get('firstName')) {
+    errors.firstName = <FormattedMessage {...messages.required} />;
+  }
+  if (!fromJS(values).get('lastName')) {
```

Comment:
These names like 'lastName' should be moved to constants as i think it will be reused all over:
```
// In constants file
const formNamesLastName = 'lastName';

export formConstants from 'constants';
if (!values[formConstants.formNamesLastName])
```